### PR TITLE
:seedling: Add release-1.8-branch-config

### PR DESCRIPTION
**Pull request by mquhuy**

### CODE REVIEW

**Review by smoshiur1237 — COMMENTED** [review]
/approve

File: README.md [hunk 1]
```diff
@@ -180,8 +180,8 @@ The following table describes which branches are tested for different test trigg
 | test suffix | CAPM3 branch | IPAM branch  | BMO branch/tag  | Keepalived tag | Ironic tag |
 | ----------- | ------------ | -----------  | --------------- | -------------- | ---------- |
 | main        | main         | main         | main            | latest         | latest     |
+| release-1-8 | release-1.8  | release-1.8  | release-0.7     | v0.6.1         | v25.0.0    |
```
Comment (tuminoid) [inline README.md:183]:
```suggestion
| release-1-8 | release-1.8  | release-1.8  | release-0.7     | v0.7.0         | v25.0.0    |
```

**Comment by tuminoid** [comment]
Can we do removing 1.5 and adding 1.8 in separate PRs please?

**Review by tuminoid — COMMENTED** [review]
/approve

**Comment by mquhuy** [comment]
@tuminoid Could you please change to /lgtm?

File: lib/common.sh [hunk 2]
```diff
@@ -289,6 +292,14 @@ elif [[ "${CAPM3RELEASEBRANCH}" = "release-1.7" ]]; then
   export IRONIC_TAG="v24.1.1"
   export BMOCOMMIT="eea41f4fbcf73c5aae7e015ad552b137ae507dbb" # tag v0.6.1
   export BMOBRANCH="${BMORELEASEBRANCH:-release-0.6}"
+elif [[ "${CAPM3RELEASEBRANCH}" = "release-1.8" ]]; then
+  export CAPM3_IMAGE=${CAPM3_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/cluster-api-provider-metal3:release-1.8"}
+  export IPAM_IMAGE=${IPAM_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/ip-address-manager:release-1.8"}
+  export BARE_METAL_OPERATOR_TAG="v0.7.0"
+  export KEEPALIVED_TAG="v0.7.0"
+  export IRONIC_TAG="v25.0.0"
+  export BMOCOMMIT="eea41f4fbcf73c5aae7e015ad552b137ae507dbb" # tag v0.6.1
```
Comment (kashifest) [inline lib/common.sh:301]: this makes little sense to me to point a v0.6.1 commit but BMO tag is v0.7.0

Comment (mquhuy) [reply lib/common.sh:301]: Ah sorry. I didn't notice the comment :sweat_smile: Changing.

File: lib/common.sh [hunk 3]
```diff
@@ -289,6 +292,14 @@ elif [[ "${CAPM3RELEASEBRANCH}" = "release-1.7" ]]; then
   export IRONIC_TAG="v24.1.1"
   export BMOCOMMIT="eea41f4fbcf73c5aae7e015ad552b137ae507dbb" # tag v0.6.1
   export BMOBRANCH="${BMORELEASEBRANCH:-release-0.6}"
+elif [[ "${CAPM3RELEASEBRANCH}" = "release-1.8" ]]; then
+  export CAPM3_IMAGE=${CAPM3_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/cluster-api-provider-metal3:release-1.8"}
+  export IPAM_IMAGE=${IPAM_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/ip-address-manager:release-1.8"}
+  export BARE_METAL_OPERATOR_TAG="v0.7.0"
+  export KEEPALIVED_TAG="v0.7.0"
+  export IRONIC_TAG="v25.0.0"
+  export BMOCOMMIT="eea41f4fbcf73c5aae7e015ad552b137ae507dbb" # tag v0.7.0
```
Comment (kashifest) [inline lib/common.sh:301]: I believe you need to change the commit hash to point to the v0.7.0 tag actually, now this and the tag v0.6.1 few lines up are pointing to the same commit hash.

Comment (mquhuy) [reply lib/common.sh]: Ah. Yes, sorry. I misunderstood this

File: lib/common.sh [hunk 4]
```diff
@@ -289,6 +292,14 @@ elif [[ "${CAPM3RELEASEBRANCH}" = "release-1.7" ]]; then
   export IRONIC_TAG="v24.1.1"
   export BMOCOMMIT="eea41f4fbcf73c5aae7e015ad552b137ae507dbb" # tag v0.6.1
   export BMOBRANCH="${BMORELEASEBRANCH:-release-0.6}"
+elif [[ "${CAPM3RELEASEBRANCH}" = "release-1.8" ]]; then
+  export CAPM3_IMAGE=${CAPM3_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/cluster-api-provider-metal3:release-1.8"}
+  export IPAM_IMAGE=${IPAM_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/ip-address-manager:release-1.8"}
+  export BARE_METAL_OPERATOR_TAG="v0.7.0"
```
Comment (tuminoid) [inline lib/common.sh:298]: We don't have this yet

**Review by tuminoid — COMMENTED** [review]
If the commit points to RC, the tags should too. They cannot point to different things, especially to ones that do not exist.

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Review by smoshiur1237 — COMMENTED** [review]
LGTM

**Comment by tuminoid** [comment]
/override metal3-centos-e2e-integration-test-release-1-7  metal3-dev-env-integration-test-ubuntu-main

**Pull request merged**